Automattic/wp-calypso
# Settings: re-enable Anti-spam upsell and update its copy

#### Changes proposed in this Pull Request

* This upsell was disabled because its copy promoted Jetpack Personal which is a plan we don't offer anymore. This PR re-enables the upsell and update its copy.

#### Testing instructions

* Run this PR.
* Select a site with Jetpack Free that doesn't have Akismet or Jetpack Anti-spam.
* Visit `/settings/security/:site`.
* Verify that you see an upsell (see capture below) in the `Anti-spam` panel.
* Verify that the upsell's copy look like the one in the capture.
* Click the upsell.
* Verify that you end up in checkout with Jetpack Anti-spam in the cart.
* Do the same with a site that has either Jetpack Security or Jetpack Complete.
* Verify that you don't see the upsell.

Fixes 1196341175636977-as-1196942190681387

#### Demo
![image](https://user-images.githubusercontent.com/3418513/95103914-9aea1980-070b-11eb-8158-39ee8abf6835.png)
